althea_types/src/interop.rs

```diff
@@ -516,10 +518,42 @@ pub struct OperatorUpdateMessage {
     /// settings for the device bandwidth shaper
     #[serde(default = "default_shaper_settings")]
     pub shaper_settings: ShaperSettings,
-    /// Updated contact info from ops tools
+    // Updated contact info from ops tools
+    #[serde(
+        serialize_with = "data_serialize",
+        deserialize_with = "data_deserialize"
+    )]
     pub contact_info: Option<ContactType>,
 }
 
+/// Serializes slice of data as "UNFORMATTED DATA" format required
+/// by Ethereum JSONRPC API.
+///
+/// See more https://github.com/ethereum/wiki/wiki/JSON-RPC#hex-value-encoding
```

Comment:
these comments need to be updated to say that we're just serializing as a string becuase of bincode and ANY values